### ISSUE TITLE
 ObjectUnderTest should be the first field of a test class

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#1146]
In case a test class defines an object under test / SUT as a field, that field should be placed before other fields.

Following should trigger a warning:
```c#
public class MyObjectTests
{
    private int someValue;
    private MyObject objectUnderTest;
}
```
It should be fixed to:
```c#
public class MyObjectTests
{
    private MyObject objectUnderTest;
    private int someValue;
}
```
